rest-api/src/main/resources/paths/activityevents/v1_activityevents.yml

```diff
@@ -0,0 +1,28 @@
+get:
+    operationId: getSelfActivityEvents
```

Comment:
I think it's sufficient here to say getActivityEvents, and only put it on the ForConsentedUsersApi. And the other method on the ParticipantsApi. But if you want to have an ActivityEventsApi, the other methods are getUsersWhatever, so I would go with getUsersActivityEvents. (getParticipantReportRecords should have been getUsersReportRecords as well).